### DISCUSSION TITLE
theme.json custom CSS tutorial

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#93] [Topic Ideas]
WordPress 6.2 will allow developers to create per-block CSS via `theme.json`: https://github.com/WordPress/gutenberg/pull/46571

This topic idea is to walk theme authors through the process of using this feature, compare it with using stylesheets, and its relationship with the Styles interface in the editor. It will provide code examples for creating CSS to handle use cases that the built-in design tools are unable to.

**Comment by juanmaguitar** [comment]
Pinging @kathrynwp in case you're interested in collaborating (or authoring) on this type of content.

**Comment by kathrynwp** [comment]
I'd certainly be interested in collaborating with @justintadlock if I could be useful. I think I'd learn a lot in the process too, and I could carry that forward to my work in the support forums and online workshops.

**Reply by justintadlock** [reply]
I haven't started exploring this yet (plan to next week). But, I'm open to hearing about any ideas you have. I'll ping you to discuss.

**Comment by bph** [comment]
This discussion is now locked. Please continue discussion on the issue 

- #97

**Comment by bph** [comment]
Closed with published article: 
[Per-block CSS with theme.json](https://developer.wordpress.org/news/2023/04/per-block-css-with-theme-json/)
